tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -98,10 +98,11 @@ message TableDescProto {
   required string table_name = 2;
   optional string path = 3;
   required TableProto meta = 4;
-  required SchemaProto schema = 5;
+  optional SchemaProto schema = 5;
```

Comment:
If we can ensure that the TableDescs of all normal tables have Schema, we can use its ``optional`` constraint to indicate either self-describing or not.